ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/KsqlResource.java

```diff
@@ -229,6 +252,7 @@ public Response handleKsqlStatements(
       return ErrorResponseUtil.generateResponse(
           e, Errors.badRequest(e));
     } catch (final Exception e) {
+      e.printStackTrace();
```

Comment:
Best to remove this

Reply:
woops, yea i'll get rid of it, left over from my debugging